src/social/components/Feed/index.js

```diff
@@ -34,6 +34,11 @@ function getAdjustedPostTarget(
   return { targetType, targetId };
 }
 
+function reloadPage() {
+  // TODO: Replace with proper feed replace
```

Comment:
did you forget this or will you do it in another PR?

Reply:
I don't have time to do it. It is something for the future devs.